ui/helpers/utils/export-utils.js

```diff
@@ -1,19 +1,13 @@
 import { getRandomFileName } from './util';
 
-export function exportAsFile(filename, data, type = 'text/csv') {
+export async function exportAsFile(filename, data, type = 'text/csv') {
```

Comment:
no longer needs to be async

Reply:
6e13ea84285f0db5eb268ef52bab6a2b5cec49b3